### PR TITLE
RHDEVDOCS 6200 Release Notes for Pipelines 1.15.2

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

The PR is into pipelines-docs-1.15 . This is intentional.
If possible please also cherrypick to pipelines-docs-main. However this module is not included in any assembly in pipelines-docs-main.

Issue:

RHDEVDOCS 6200

Link to docs preview:

https://83476--ocpdocs-pr.netlify.app/openshift-pipelines/latest/about/op-release-notes.html#release-notes-1-15-2_op-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mramendi** [comment]
https://issues.redhat.com/browse/RHDEVDOCS-6200

**Review by vikram-raj — COMMENTED** [review]
LGTM

**Review by ppitonak — APPROVED** [review]

File: modules/op-release-notes-1-15.adoc [hunk 1]
```diff
@@ -472,3 +472,19 @@ With this update, {pipelines-title} General Availability (GA) 1.15.1 is availabl
...
+
+* Before this update, in the web console, sorting of pipeline runs and task runs by creation time did not work. Sorting of pipelines by last run time also did not work. With this update, the sorting works correctly.
+
+* Before this update, in the web console, if you enabled the {pipelines-shortname} console plugin, when starting a pipeline you could not select the storage class for the volume claim template, because the *StorageClass* list was not present in the *VolumeClaimTemplate* options. With this update, you can select the storage class for the volume claim template.
+
+* Before this update, if you used {pipelines-as-code}, in the *Repository details* page of the web console the list of pipeline runs did not display correctly. With this update, the list of pipeline runs displays correctly.
```
Comment (skopacz1) [inline modules/op-release-notes-1-15.adoc:490]: The pipelines as code attribute doesn't resolve in the rendered preview

Comment (skopacz1) [inline modules/op-release-notes-1-15.adoc:490]: I would remove this second comma, or shift it to later in the sentence:
```suggestion
* Before this update, if you used {pipelines-as-code} in the *Repository details* page of the web console, the list of pipeline runs did not display correctly. With this update, the list of pipeline runs displays correctly.
```

**Review by skopacz1 — COMMENTED** [review]
A few nits, otherwise LGTM

File: modules/op-release-notes-1-15.adoc (hunk 1 above)
Comment (mramendi) [reply modules/op-release-notes-1-15.adoc:490]: Unfortunately removing or moving the second comma makes the sentence incorrect. Using PAC is something you do in general, and not in the web console (usually). But if you do use PAC, then in the web console the list does not display correctly.

Comment (skopacz1) [reply modules/op-release-notes-1-15.adoc]: I see what you mean, now that I look at this sentence again it makes more sense. Feel free to keep it as-is, but one other suggestion might be to switch around part of this sentence (assuming it remains accurate):
`Before this update, if you used {pipelines-as-code}, the list of pipeline runs did not display correctly in the *Repository details* page of the web console.`

But it's up to you, probably fine either way.

**Review by lokanandaprabhu — APPROVED** [review]
/lgtm

**Pull request merged**